Styles/DarkStyles.tsx

```diff
@@ -0,0 +1,46 @@
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#090b38",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+
+  //header
```

Comment:
No need to comment the obvious, i am aware some of it is from previous PR but can be changed